site/styles/_availability-tags.scss

```diff
@@ -71,3 +71,25 @@ s72-plan-label {
   font-size: 12px;
   text-align: center;
 }
+
+s72-live-lozenge {
+  .live-lozenge {
+    background-color: var(--live-lozenge-background-color);
+    border-radius: 100px;
+    font-size: 12px;
+    line-height: 30px;
+    padding: 0 12px;
+    white-space: nowrap;
+    width: fit-content;
+
+    &.is-live {
+      background-color: var(--live-lozenge-live-background-color);
+    }
+
+    .fa-circle {
+      @extend .fa;
+
+      margin-right: 8px;
+    }
```

Comment:
Should this be a before pseudo element mixin that makes a circle instead? e.g. `content: "", border: 50%, display: inline-block etc`

Reply:
Could do... I just rolled with circle icon because it's there in our custom font (`fa-` is a red herring).

Reply:
@matthewthomsonnz @additiverich Can I get some votes... custom circle or fa-circle?  Core-template has a mixture of custom made icons and fa-icons so hard to argue one way or the other based on consistency.

Reply:
[I used custom circle](https://github.com/shift72/core-template/pull/427/files#diff-1690b8cb885e67b64253816006cb361c5af104c3073fe62ebfb567c91815dcd5R72-R80) and it aligned well. Either or I don't mind both solutions work.

Whichever choice can you make a reusable mixin and call it here? As I'll add it to my branch for consistency

Reply:
Mixin added!